Addressed Github issue #7843 (mute on unthrottle)

MooglyGuy opened the pull request:
Intended commit/whatsnew message:

-ui: Added new option, -unthrottle_mute / -utm, to optionally mute audio output when running unthrottled. Addresses Github issue #7843. [Ryan Holtz]

cuavas commented on src/emu/sound.h:
```diff
@@ -740,6 +740,7 @@ class sound_manager
 	static constexpr u8 MUTE_REASON_UI = 0x02;
 	static constexpr u8 MUTE_REASON_DEBUGGER = 0x04;
 	static constexpr u8 MUTE_REASON_SYSTEM = 0x08;
+	static constexpr u8 MUTE_REASON_UNTHROTTLE = 0x10;
```
I don’t think this is necessary – you can use `ui_mute` since that’s there for the UI manager’s use while the machine is running.

cuavas reviewed (commented):
Should the mute-on-unthrottled setting also apply to fast-forward?

MooglyGuy commented:
I have addressed all of these issues, and pushed the fixes to my branch, but I'm unclear on how to actually update this PR with those changes. What do I do?

cuavas commented:
The PR gets updated when you push to your branch and people following are notified.
